Spigot-Server-Patches/0440-Fix-MC-167561-All-players-can-make-a-dog-stand-up-or.patch

```diff
@@ -0,0 +1,23 @@
+From 0079d030b962067cce26825e127012132f6f98d5 Mon Sep 17 00:00:00 2001
+From: Callahan <mr.callahhh@gmail.com>
+Date: Fri, 14 Feb 2020 10:19:06 -0600
+Subject: [PATCH] Fix MC-167561 - All players can make a dog stand up or sit
+ down
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityWolf.java b/src/main/java/net/minecraft/server/EntityWolf.java
+index db15d5e0a..099ebb006 100644
+--- a/src/main/java/net/minecraft/server/EntityWolf.java
++++ b/src/main/java/net/minecraft/server/EntityWolf.java
+@@ -295,7 +295,7 @@ public class EntityWolf extends EntityTameableAnimal {
+                 if (!(item instanceof ItemDye)) {
+                     boolean flag = super.a(entityhuman, enumhand);
+ 
+-                    if (!flag || this.isBaby()) {
++                    if ((!flag || this.isBaby()) && this.i(entityhuman)) { // Paper - Fix MC-167561
```

Comment:
But, if reverting the entire method isn't in the core team's interest, at least make an obfhelper for the `isOwner` call here